src/libraries/System.Security.Cryptography/src/Resources/Strings.resx

```diff
@@ -945,4 +1005,7 @@
   <data name="Cryptography_X509_PfxWithoutPassword_ProblemFound" xml:space="preserve">
     <value>There was a problem with the PKCS12 (PFX) without a supplied password. See https://go.microsoft.com/fwlink/?linkid=2233907 for more information.</value>
   </data>
+  <data name="Cryptography_SlhDsaInvalidAlgorithmHandle" xml:space="preserve">
+    <value>The specified EVP_PKEY handle is not a known SLH-DSA algorithm.</value>
+  </data>
```

Comment:
Since you've had multiple PRs put these at the end, and insert the big unnecessary comment block at the top, I assume you're using the VS RESX form editor.  Just edit it as an XML/text file :)

Reply:
Ugh I don't know why the resx generator doesn't have an option to sort.